### ISSUE TITLE
Prediction labels

### ISSUE THREAD

**Issue by taewookim** [sebastianbk/finetuned-resnet50-keras#5]
when you make a prediction, # of outputs matches the number of classes from train / validation.

Question.. are the order of the predictions the same alphabetical order of the train/validation directory?>

in another words... if the directory had

> train / cat
> train / dog
> train / monkey
> valid / cat
> valid / dog
> valid / monkey

And the predictios are [  0.9, 0.03, 0.07]... is it safe to assume that
0.9 = cat
0.03 =  dog
0.07 = monkey

?

**Comment by jparedesj** [comment]
Hello

I think your assumption is true
